gtclang/src/gtclang/Driver/CompilerInstance.cpp

```diff
@@ -93,6 +93,8 @@ clang::CompilerInstance* createCompilerInstance(llvm::SmallVectorImpl<const char
   // Set the root where system headers are located.
   ccArgs.push_back("-internal-isystem");
   ccArgs.push_back(GTCLANG_CLANG_RESSOURCE_INCLUDE_PATH "/../../../../include/c++/v1/");
+  ccArgs.push_back("-internal-isystem");
```

Comment:
this arg is pushed back twice. Is that intended?

Reply:
Yes. It's like `-I` but internally, so the path follows that argument each time.